src/Symfony/Bridge/Monolog/Command/ServerLogCommand.php

```diff
@@ -153,6 +155,18 @@ private function displayLog(OutputInterface $output, int $clientId, array $recor
         $logBlock = sprintf('<bg=%s> </>', self::BG_COLOR[$clientId % 8]);
         $output->write($logBlock);
 
+        $record = new LogRecord(
+            $record['datetime'],
+            $record['channel'],
+            Level::fromValue($record['level']),
+            $record['message'],
+            // We wrap context and extra, because they have been already dumped.
+            // So they are instance of Symfony\Component\VarDumper\Cloner\Data
+            // But LogRecord expects array
```

Comment:
iterator_to_array to the rescue?

Reply:
or better: `->getValue()`

Reply:
It cannot works (seen with @nicolas-grekas on slack)

TL;DR:
`$record` is now typed, with `array`. Before we used to put an instance of `Data` so it used to work. Now, we wrap it un an array, and we unwrap it in `ConsoleFormatter::format()` so `ConsoleFormatter::dumpData()` can work as expected

But I will update it a bit, to avoid passing this wild option `from_server_log`